modules/meeting/spec/support/pages/meetings/show.rb

```diff
@@ -69,6 +69,23 @@ def expect_date_time(expected)
         .to have_content("Time: #{expected}")
     end
 
+    def expect_link_to_location(location)
+      within(meeting_details_container) do
+        expect(page).to have_link location
+      end
+    end
+
+    def expect_plaintext_location(location)
+      within(meeting_details_container) do
+        expect(page).not_to have_link location
+        expect(page).to have_text(location)
+      end
+    end
+
+    def meeting_details_container
+      find('.meeting.details')
+    end
```

Comment:
I'm aware the meetings module is due for a total makeover so extracting this might aid the do-over just a smidge :)